examples/create-react-app-with-flow/flow-typed/npm/material-ui_v1.x.x.js

```diff
@@ -121,7 +121,7 @@ declare module "material-ui/ButtonBase/ButtonBase" {
     disableRipple?: boolean,
     disabled?: boolean,
     focusRipple?: boolean,
-    keyboardFocusedClassName?: string,
+    focusVisibleedClassName?: string,
```

Comment:
@oliviertassinari Looks like there is a typo here, it should be `focusVisibleClassName`.

Reply:
@lucaas Oops, thank you for the review, the `flow-typed/npm/material-ui_v1.x.x.js` wasn't meant to be changed. It's generated by the flow-typed tools. Do you want to fix it? :)

Reply:
Sorry, not sure I can fix it. Was just browsing through the recent changes and stumbled upon it. Good work on the v1 release so far, looks very promising.